internal/storage/resource_storage_pool_test.go

```diff
@@ -214,6 +214,8 @@ func TestAccStoragePool_target(t *testing.T) {
 	poolName := petname.Generate(2, "-")
 	driverName := "dir"
 
+	clusterMemberNames := make(map[string]struct{}, 10) // It is unlikely, that acceptance tests are executed against clusters > 10 nodes.
```

Comment:
As far as I can tell, are the acceptance tests executed without concurrency, so accessing this from the check functions should be fine. If we want to be extra secure, we could add some mutex.